### PR TITLE
Governor: Reworking manual token handling

### PR DESCRIPTION
**Pull request by bruce-riley**
This PR changes the way tokens are configured in the governor. It separates the list of manually added tokens from the auto generated ones to make it easier to update the generated list.

This PR also regenerates the token list, so reviewers should review that carefully.

Change-Id: Ic674fb95e89b9cb94c4fe1f9acbd0c5b114f2ebb

### CODE REVIEW

**Review by ckeun — APPROVED** [review]

**Review by ckeun — APPROVED** [review]

**Review by ckeun — APPROVED** [review]

File: node/hack/governor/README.md [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
 To verify that the Coin Gecko query still works with the new token list, do:
 ```
 go run check_query.go
 ```
 
-You can then commit the updated version of node/pkg/governor/tokens.go.
+Before committing the generated file, you should run the governor tests and ensure that they pass:
+```
+cd wormhole/node/pkg/governor
+go test
```
Comment (claudijd) [inline node/hack/governor/README.md:36]: These are run in CI as well, right?  Just to keep people honest?

File: node/pkg/governor/manual_tokens.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		tokenConfigEntry{chain: 11, addr: "0000000000000000000000000000000000000000000100000000000000000082", symbol: "KSM", coinGeckoId: "kusama", decimals: 8, price: 41.71},
+		tokenConfigEntry{chain: 12, addr: "0000000000000000000000000000000000000000000100000000000000000002", symbol: "DOT", coinGeckoId: "polkadot", decimals: 8, price: 6.48},
+		tokenConfigEntry{chain: 13, addr: "0000000000000000000000005fff3a6c16c2208103f318f4713d4d90601a7313", symbol: "KLEVA", coinGeckoId: "kleva", decimals: 8, price: 0.086661},
+		tokenConfigEntry{chain: 13, addr: "0000000000000000000000005096db80b21ef45230c9e423c373f1fc9c0198dd", symbol: "WEMIX", coinGeckoId: "wemix-token", decimals: 8, price: 1.74},
+		tokenConfigEntry{chain: 13, addr: "0000000000000000000000005c74070fdea071359b86082bd9f9b3deaafbe32b", symbol: "KDAI", coinGeckoId: "dai", decimals: 8, price: 1.00},
+		tokenConfigEntry{chain: 15, addr: "0000000000000000000000000000000000000000000000000000000000000000", symbol: "NEAR", coinGeckoId: "near", decimals: 8, price: 3.85},
```
Comment (claudijd) [inline node/pkg/governor/manual_tokens.go:17]: What's the background context on the all-zeros NEAR address?  Is that legitimately the token address or is some sort of shim?

File: node/hack/governor/README.md (hunk 1 above)
Comment (claudijd) [reply node/hack/governor/README.md:36]: nvmd, answered my own question, yes.

**Review by claudijd — APPROVED** [review]

File: node/pkg/governor/manual_tokens.go (hunk 2 above)
Comment (ckeun) [reply node/pkg/governor/manual_tokens.go:17]: i believe this is the hex representation of the NEAR native token address that you would expect to see in the signed vaa. Similarly, this is also the case for the ALGO native token address. @jumpsiegel may have more insight into the whys. 😉

**Review by njkumr — APPROVED** [review]
lgtm

**Pull request merged**